ytfzf

```diff
@@ -975,16 +980,13 @@ format_user_selection () {
 		[ -z "$surl" ] && continue # to ignore the separators
 		#literally just check if there is :// to make sure it's a url (just in case the url doesn't use https or http)
 		#peertube urls can differ a lot
-		case $surl in
-			*'://'*) selected_urls=$selected_urls$new_line$surl ;;
-			*)
-				case ${#surl} in
+		case ${#surl} in
 					# youtube video
-					11) selected_urls=$selected_urls$new_line'https://www.youtube.com/watch?v='$surl ;;
-					34) selected_urls=$selected_urls$new_line'https://www.youtube.com/playlist?list='$surl ;;
-					*) continue ;;
-				esac
-				;;
+			11) selected_urls=$selected_urls$new_line'https://www.youtube.com/watch?v='$surl ;;
+			34) selected_urls=$selected_urls$new_line'https://www.youtube.com/playlist?list='$surl ;;
+			36)
```

Comment:
oh so for peertube the pre-url part  is not consistent?

Reply:
in order to do thumbnails, i had to store the video id for the file path to download the thumbnail to. I am also using the video id here which will always be a UUID of 36 characters.